### PR TITLE
feat: add complexity into the graphql operation span

### PR DESCRIPTION
**Pull request by obmarg**

### CODE REVIEW

File: crates/engine/src/response/mod.rs [hunk 1]
```diff
@@ -44,6 +44,7 @@ pub(crate) enum Response<OnOperationResponseHookOutput> {
 
 pub(crate) struct ExecutedResponse<OnOperationResponseHookOutput> {
     operation: Arc<CachedOperation>,
+    attributes: GraphqlOperationAttributes,
```
Comment (Finistere) [inline crates/engine/src/response/mod.rs:47]: `operation_attributes`?

**Review by Finistere — APPROVED** [review]

File: crates/engine/src/prepare/operation/mod.rs [hunk 2]
```diff
@@ -94,3 +102,17 @@ fn mutation_not_allowed_with_safe_method<OnOperationResponseHookOutput>() -> Res
         )],
     )
 }
+
+fn validate_prepared_operation(operation: &PreparedOperation, settings: &Settings) -> PrepareResult<()> {
+    let ComplexityControl::Enforce { limit, .. } = settings.complexity_control else {
+        return Ok(());
+    };
+    let Some(complexity) = operation.complexity else {
+        return Ok(());
```
Comment (Finistere) [inline crates/engine/src/prepare/operation/mod.rs:111]: Shouldn't that be an `InternalError` or smth?

Comment (obmarg) [reply crates/engine/src/prepare/operation/mod.rs:111]: Although I should at the least add an error message in probably.

Comment (Finistere) [reply crates/engine/src/prepare/operation/mod.rs:111]: yeah but silently having a problem sounds worse to me. At least there should be a `tracing::error!` in that case

**Pull request merged**